code/iomethods.py

```diff
@@ -518,5 +531,99 @@ def decode_time_from_string(timeString):
     print 'Error decoding time string: string does not match a predefined time format'
     return 0
     # **********************************************************************************************************************
+def write_np_array_to_ncdf(lon, lat, inputData, fileName, dirName, globalAttrDict, dimensionsDict, variablesDict):
```

Comment:
Interesting approach. I was thinking to pass dictionaries like 
lonDict = {"longitude_type": ("double", ("longitude",), "units": "degrees_east", "long_name": "Longitude", "values": lon}
latDict, timeDict, varDict, etc. @cgoodale Thoughts